docs/cli/streaming-logs.md

```diff
@@ -83,7 +83,38 @@ timer per step: 0.00873
 Combining both subcommand `log` and `kill` will allow you to quickly adjust or 
 terminate simulations to achieve the desired simulation configurations.
 
-At the end of the simulation, the log stream will pause without new messages. In 
-this case, you can exit the log stream by simply using `Ctrl+C`, which stops the 
-log monitoring without affecting the simulation or its outputs.
+At the end of the simulation, the log stream will automatically close, indicating
+that there are no new messages. This automatic closure of the stream does not 
+affect the simulation or its outputs, and no further action is required from the
+user.
 
+In addition to the automatic closure of the log stream, the `inductiva logs`
+command supports several modes for redirecting output:
+
+- **Redirect stdout to a file**: You can redirect stdout to a file using the 
+`1>` operator. For example:
```

Comment:
```suggestion
- **Redirect stdout to a file**: You can redirect stdout (file descriptor 1) to a file using the 
redirection operator (`>`). This will redirect the entire content of the stream to a file. Example:
```